src/checkout-buttons/strategies/braintree/braintree-paypal-button-strategy.ts

```diff
@@ -61,7 +62,7 @@ export default class BraintreePaypalButtonStrategy implements CheckoutButtonStra
         };
 
         return Promise.all([
-            this._braintreeSDKCreator.getPaypalCheckout({currency: currency?.code}, (paypalCheckoutInstance: PaypalClientInstance) => this._renderButtons(paypalCheckoutInstance)),
+            this._braintreeSDKCreator.getPaypalCheckout({currency: currency?.code, paypalMerchantId}, (paypalCheckoutInstance: PaypalClientInstance) => this._renderButtons(paypalCheckoutInstance)),
```

Comment:
Do we need to check if this can be undefined?

Reply:
we check it inside getPaypalCheckout function